output/outputFormat.go

```diff
@@ -143,6 +143,14 @@ var (
 		Rel:         "alternate",
 	}
 
+	WebAppManifestFormat = Format{
+		Name:        "Web App Manifest",
```

Comment:
I suggest name `WebAppManifest` which should make life easier.

Also, could you add this type to the `DefaultFormats`, also I suspect there will be a test failure or two.

Reply:
OK, sorry for the confusion. The type should follow the naming standard of the others (that is a discussion for another time :-); I meant the `Name`,  which is now `Web App Manifest`.